apps/frontend/src/libs/components/table/styles.module.css

```diff
@@ -0,0 +1,38 @@
+.table-container {
```

Comment:
Do we need a table container at all in a common component?  It looks like this should be added on the main page, where you place your component. By default, the table will take 100% of it's container and we will add wrappers and containers in the place where we use this table.